modules/claims_api/app/models/claims_api/auto_established_claim.rb

```diff
@@ -64,5 +65,23 @@ def set_md5
     def status_from_phase(*)
       status
     end
+
+    def file_name
```

Comment:
I feel like these same methods are duplicated all over the place in claims api module...could you refactor?

Reply:
Refactored.